### PR TITLE
📖 Bento Youtube

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: extensions/amp-youtube/1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,564 @@
...
+  </tr>
+</table>
+
+### Interactivity and API usage
+
+The `bento-youtube` component API is accessible by including the following script tag in your document:
+
+```javascript
+await customElements.whenDefined('bento-video');
+const api = await carousel.getApi();
```
Comment (kristoferbaxter) [inline extensions/amp-youtube/1.0/README.md:184]: Typo? carousel?

**Review by kristoferbaxter — APPROVED** [review]
Left one nit.

File: extensions/amp-youtube/1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,564 @@
...
+```css
+.custom-styles {
+  height: 320px;
+  width: 180px;
+}
+```
+
+## Props
+
+<table>
```
Comment (sebastianbenz) [inline extensions/amp-youtube/1.0/README.md:394]: Please don't use tables

File: extensions/amp-youtube/1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,564 @@
...
+    />
+    <style>
+      bento-youtube {
+        width: 320px;
+        height: 180px;
+      }
+    </style>
+  </head>
+  <body>
+    <bento-youtube data-videoid="dQw4w9WgXcQ"></bento-youtube>
```
[the remark this answers is not in the document]
Comment (alanorozco) [reply extensions/amp-youtube/1.0/README.md:67]: Not at the moment.

**Review by sebastianbenz — APPROVED** [review]
Thanks!

**Pull request merged**